components/footer/footer-link.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import useTheme from '../use-theme'
+
+interface Props {
+  target?: React.AnchorHTMLAttributes<HTMLAnchorElement>['target']
+  href: string
+}
+
+type NativeAttrs = Omit<React.HTMLAttributes<any>, keyof Props>
+export type FooterLinkProps = Props & NativeAttrs
+
+const FooterLink: React.FC<React.PropsWithChildren<FooterLinkProps>> = ({
+  href,
+  children,
+  target,
+  ...props
+}) => {
+  const theme = useTheme()
+  return (
+    <li {...props}>
+      <a href={href} target={target || '_self'}>
```

Comment:
Should we allow users to customize the `Element` (`a`),  or use the `Link` component directly.
If children contains the `a` element, we should just use it instead of the template here.

Reply:
yeah this is a good case to consider ✅ on it.